scripts/run_example

```diff
@@ -4,38 +4,72 @@ readonly SCRIPTS_DIR="$(dirname "$(readlink -f "$0")")"
 # shellcheck source=scripts/common
 source "$SCRIPTS_DIR/common"
 
-while getopts ":hC" opt; do
-    case "$opt" in
-    h)
-        echo -e "Usage: $0 [-c] [-h] [-C opts] example_name
-  -h    Print this message
-  -C    Pass subsequent options to example client"
-        exit 0
-        ;;
-    C)
-        echo "Passing remaining arguments to client"
-        break
-        ;;
-    \?)
-        echo "Invalid option -${OPTARG}"
-        exit 1
-        ;;
-    esac
+language="rust"
```

Comment:
Can we remove the `run` scripts within each example subfolder now? e.g. https://github.com/project-oak/oak/blob/master/examples/chat/run and similar. Though I'm not sure why they are there in the first place, so let me know if I'm missing something (@daviddrysdale perhaps you know more about this?)

Reply:
I don't remember why they were left in place when example running got commonized (b280b47a5a636b2cd6972290355671cdaa9f9b68); maybe because other places (docs?) referenced them?

Reply:
Deleted scripts and references in docs